peerconnection.go

```diff
@@ -1768,13 +1790,17 @@ func (pc *PeerConnection) startRTP(isRenegotiation bool, remoteDesc *SessionDesc
 	trackDetails := trackDetailsFromSDP(pc.log, remoteDesc.parsed)
 	if isRenegotiation {
 		for _, t := range currentTransceivers {
+			// Is `SendOnly` transceiver or receiver has no track. Receivers with no track
+			// might get a track later when `Receive()` is called on the receiver.
 			if t.Receiver() == nil || t.Receiver().Track() == nil {
 				continue
 			}
 
 			t.Receiver().Track().mu.Lock()
 			ssrc := t.Receiver().Track().ssrc
 
+			// If receiver already exists for track ssrc, set id/label.
+			// Why do we do this? Is there a case where the track id/label changes?
```

Comment:
In this case, if `t.Receiver().Track().label != details.label`, should we remove the track from `stream[t.Receiver().Track().label]` and add it to the stream with new label?

Reply:
I'm not sure 100%, but it seems a SSRC can be reused with a different id/label. Is that true?